voidcrew/code/__HELPERS/game.dm

```diff
@@ -19,3 +19,20 @@
 			to_chat(character, "<h2>Tag your ship with KOS to fight NT-C, SYN-C, and KOS ships</h2>")
 		else
 			return
+
+/proc/AnnounceArrival(mob/living/carbon/human/character, rank, obj/structure/overmap/ship/simulated/ship)
```

Comment:
TG update is gonna kinda screw this over since commenting out is still considered a TG edit, which sucks.